types/test/typescriptDocs/ModelInitWithAttributesOf.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * Keep this file in sync with the code in the "Using with `AttributesOf`" section in typescript.md
```

Comment:
This note is only for the contributors of Sequelize right? Even though, we should put the complete path to the file

Reply:
Done, as well as the other files like this